packages/coinjoin/src/backend/CoinjoinBackendClient.ts

```diff
@@ -192,7 +192,10 @@ export class CoinjoinBackendClient {
             case 204:
                 return { status: 'up-to-date' };
             case 200: {
-                const result: { bestHeight: number; filters: string[] } = await response.json();
+                const result: { bestHeight: number; filters: string[] } = await response
+                    .json()
+                    .then(r => patchResponse(r));
```

Comment:
nitpicky nitpick: 
```suggestion
                    .then(patchResponse);
```

Reply:
I know it's not the same in terms of types, not sure if it's here because of that. No need to change anything here...